src/pybind/mgr/dashboard/services/proto/gateway_pb2_grpc.py

```diff
@@ -0,0 +1,747 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
+import grpc
+
+from . import gateway_pb2 as gateway__pb2
```

Comment:
The way to make this import be properly auto-generated is [described here](https://grpc.io/docs/languages/python/basics/#generating-grpc-interfaces-with-custom-package-path).

The above was manually modified.

It should be something like:

```sh
$ python -m grpc_tools.protoc -I./services/proto \
  --python_out=. --grpc_python_out=. \
  services/proto/gateway.proto
```